code/game/sound.dm

```diff
@@ -220,20 +220,36 @@
 				S = pick('sound/voice/alien_queen_command.ogg','sound/voice/alien_queen_command2.ogg','sound/voice/alien_queen_command3.ogg')
 			// Human
 			if("male_scream")
-				S = pick('sound/voice/human_male_scream_1.ogg','sound/voice/human_male_scream_2.ogg','sound/voice/human_male_scream_3.ogg','sound/voice/human_male_scream_4.ogg','sound/voice/human_male_scream_5.ogg','sound/voice/human_male_scream_6.ogg',5;'sound/voice/human_male_scream_joke1.ogg',5;'sound/voice/human_male_scream_joke2.ogg',5;'sound/voice/human_male_scream_joke3.ogg',5;'sound/voice/human_male_scream_joke4.ogg')
+				S = pick('sound/voice/human_male_scream_1.ogg','sound/voice/human_male_scream_2.ogg','sound/voice/human_male_scream_3.ogg','sound/voice/human_male_scream_4.ogg','sound/voice/human_male_scream_5.ogg','sound/voice/human_male_scream_6.ogg')
```

Comment:
why are you removing the weightings from these options?

Reply:
Because I’m removing all the “joke” sounds that currently are weighted to play only on rare occasions.  None of the sounds here are meant to be rare meme sounds